tests/unit/fem/test_quadinterpolator.cpp

```diff
@@ -194,11 +240,11 @@ static bool testQuadratureInterpolator(const int dim,
 TEST_CASE("QuadratureInterpolator", "[QuadratureInterpolator]")
 {
    const auto d = GENERATE(2,3); // dimension
-   const auto p = GENERATE(range(1,8)); // element order
-   const auto q = GENERATE_COPY(range(p+1,10)); // quadrature order
+   const auto p = GENERATE(range(1,7)); // element order, 1 <= p < 7
+   const auto q = GENERATE_COPY(p+1,p+2); // 1D quadrature points
```

Comment:
Does the last fallback with specific `GRID` is tested for the `det` kernel?

Reply:
I think it is tested with a few combinations: `p=5,q=7`, `p=6,q=7`, and `p=6,q=8` -- in all these cases `q=Q1D>6` which set `SMEM=0` and so the kernel uses `GRID=128`.